packages/plugin-pack/sources/commands/pack.ts

```diff
@@ -39,7 +47,8 @@ export default class PackCommand extends BaseCommand {
     if (!workspace)
       throw new WorkspaceRequiredError(this.context.cwd);
 
-    const target = ppath.resolve(workspace.cwd, toFilename(`package.tgz`));
+    const archiveName = this.out.replace('%n', prettyWorkspaceSlug(workspace));
```

Comment:
I'm fine with the slug you mentioned in the other issue (I'd however prefer `%s` as it's unambiguously a string, versus `%n` which is a number in printf formatting), but can we also add a `%v`? I think some people might want to name their archives themselves:

```
yarn pack -o my-package-%v.zip
```

Reply:
done